src/Analyzers/CSharp/Analyzers/UseIsNullCheck/CSharpUseIsNullCheckOverIsObject.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.CodeStyle;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.UseIsNullCheck
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    internal sealed class CSharpUseIsNullCheckOverIsObject : AbstractBuiltInCodeStyleDiagnosticAnalyzer
+    {
+        public CSharpUseIsNullCheckOverIsObject()
+            : base(IDEDiagnosticIds.UseIsNullOverIsObjectDiagnosticId,
+                   EnforceOnBuildValues.UseIsNullCheck,
+                   CSharpCodeStyleOptions.PreferIsNullCheckOverIsObject,
+                   CSharpAnalyzersResources.Use_is_null_check,
+                   new LocalizableResourceString(nameof(AnalyzersResources.Null_check_can_be_simplified), AnalyzersResources.ResourceManager, typeof(AnalyzersResources)))
```

Comment:
it's an interesting string to show. 

`is object` vs
`is not null`

One might technically say that the former is 'simpler' since it's clearer.  We might say `Null check can be clarified` instead.